csl-data.json

```diff
@@ -1,431 +1,452 @@
+                },
+                "edition": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "issue": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "number": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "number-of-pages": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "number-of-volumes": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "volume": {
+                    "$ref": "#/definitions/numberDefinition"
+                }
+            }
+        },
+        "stringFields": {
+            "properties": {
+                "abstract": {
+                    "type": "string"
+                },
+                "annote": {
+                    "type": "string"
+                },
+                "archive": {
+                    "type": "string"
+                },
+                "archive_location": {
+                    "type": "string"
+                },
+                "archive-place": {
+                    "type": "string"
+                },
+                "authority": {
+                    "type": "string"
+                },
+                "call-number": {
+                    "type": "string"
+                },
+                "citation-label": {
+                    "type": "string"
+                },
+                "citation-number": {
+                    "type": "string"
+                },
+                "collection-title": {
+                    "type": "string"
+                },
+                "container-title": {
+                    "type": "string"
+                },
+                "container-title-short": {
+                    "type": "string"
+                },
+                "dimensions": {
+                    "type": "string"
+                },
+                "DOI": {
+                    "type": "string"
+                },
+                "event": {
+                    "type": "string"
+                },
+                "event-place": {
+                    "type": "string"
+                },
+                "first-reference-note-number": {
+                    "type": "string"
+                },
+                "genre": {
+                    "type": "string"
+                },
+                "ISBN": {
+                    "type": "string"
+                },
+                "ISSN": {
+                    "type": "string"
+                },
+                "journalAbbreviation": {
+                    "type": "string"
+                },
+                "jurisdiction": {
+                    "type": "string"
+                },
+                "keyword": {
+                    "type": "string"
+                },
+                "language": {
+                    "type": "string"
+                },
```

Comment:
Maybe it would make sense to carve out a separate definition for non-CSL variable metadata fields like "language"?